monasca-api-python/README.md

```diff
@@ -92,6 +92,7 @@ A number of environment variables can be passed to the container:
 | `KEYSTONE_ADMIN_PASSWORD` | `secretadmin` | Keystone admin account password  |
 | `KEYSTONE_ADMIN_TENANT`   | `admin`       | Keystone admin account tenant    |
 | `KEYSTONE_INSECURE`       | `false`       | Allow insecure Keystone connection |
+| `KEYSTONE_REGION_NAME`    | `undefined`   | Keystone admin account region    |
```

Comment:
Sorry to bother you again, can you remove back tics around undefined? So that operator understand on first sign that it's not variable that is set to `undefined` but it's really undefined.